src/Fallback.js

```diff
@@ -0,0 +1,28 @@
+import geolocationStates from './geolocationStates'
+
+const Fallback = ({geolocationState}) => (
+  <div className="spinnerWrapper">
+    {
+      geolocationState === geolocationStates.requested && (
+        <div className='spinner-border text-warning'></div>
+      )
+    }
+    {
+      geolocationState === geolocationStates.loading && (
+        <div className='spinner-border text-success'></div>
+      )
+    }
+    {
+      geolocationState === geolocationStates.disallowed && (
```

Comment:
I don't think we'll ever get here. If `geolocationState` is `geolocationStates.disallowed`, `<Fallback />` is not rendered.